packages/rocketchat-grant/server/authenticate.js

```diff
@@ -10,7 +10,7 @@ const setAvatarFromUrl = (userId, url) => new Promise((resolve, reject) => {
 	Meteor.runAsUser(userId, () => {
 		Meteor.call('setAvatarFromService', url, '', 'url', (err) => {
 			if (err) {
-				if (err.details.timeToReset && err.details.timeToReset) {
+				if (err.details.timeToReset) {
```

Comment:
```suggestion
				if (err.details && err.details.timeToReset) {
```

lol.. good find.. I think this was what they tried to do..

Reply:
👍 makes sense